kubernetes-manifests/frontend.yaml

```diff
@@ -88,6 +88,9 @@ spec:
           #   value: "true"
           # - name: FRONTEND_MESSAGE
           #   value: "Replace this with a message you want to display on all pages."
+          # As part of a Google Cloud demo, you can run an optional microservice called the "packaging service".
```

Comment:
nit: Placing it into main branch takes it beyond Google Cloud demo. Do we want to make it optional or change the comment here?